oadp-operator/controllers/restic.go

```diff
@@ -1,14 +1,232 @@
 package controllers
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/go-logr/logr"
 	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	"github.com/openshift/oadp-operator/pkg/common"
+	"github.com/openshift/oadp-operator/pkg/credentials"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	Restic           = "restic"
+	resticPvHostPath = "/var/lib/kubelet/pods" //TODO: make configurable with env var
+)
+
+var (
+	mountPropagationToHostContainer = v1.MountPropagationHostToContainer
 )
 
 func (r *VeleroReconciler) ReconcileResticDaemonset(log logr.Logger) (bool, error) {
 	velero := oadpv1alpha1.Velero{}
 	if err := r.Get(r.Context, r.NamespacedName, &velero); err != nil {
 		return false, err
 	}
+
+	// Define "static" portion of daemonset
+	ds := &appsv1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      Restic,
+			Namespace: r.NamespacedName.Namespace,
```

Comment:
Cool, this works. I think in other places I used `velero.Namespace` but using the reconciler namespace makes sense too